download-latest-release.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -euo pipefail
+
+RELEASE_JSON=$( curl -s "https://api.github.com/repos/matrix-org/synapse/releases" | jq '.[0]' )
+
+VERSION=$( echo $RELEASE_JSON | jq -r '.name' )
+DOWNLOAD_URL=$( echo $RELEASE_JSON | jq -r '.assets | .[] | select(.name == "debs.tar.xz") | .browser_download_url' )
+
+echo "Downloading $VERSION"
```

Comment:
It'd be useful to have some indication of where the output is.
```suggestion
echo "Downloading $VERSION to debian/incoming..."
```